README.rst

```diff
@@ -63,6 +63,10 @@ Configuring the Host Cloud
    happen with the baremetal instances booted from an empty image it speeds
    things up a bit.
 
+#. ``/etc/nova/nova.conf`` needs ``allow_duplicate_networks=true`` in the
```

Comment:
I assume this must be the default since I don't have anything set in my nova.conf and it works.  Still good to note though since I know Derek ran into it too.